qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,341 @@
+from .ATS import AlazarTech_ATS
+from .utils import TraceParameter
+from qcodes.utils import validators
+
+
+class AlazarTech_ATS9440(AlazarTech_ATS):
+    """
+    This class is the driver for the ATS9440 board
+    it inherits from the ATS base class
+    """
+    samples_divisor = 256 #32
```

Comment:
This should be 32

Reply:
I had 32 as seen but I kept getting an error that is has to be greater than 256. I'll change it back for 32 for now and will check again if I get the same error after all the changes we made.